### ISSUE TITLE
Build for OS Unix on Win Fails

### ISSUE THREAD

**Issue by ManickaP** [dotnet/corefx#41026]
Build of the repo for Unix OS fails with:
```
ODBC32' does not exist in the namespace 'System.Data.Odbc' 
(are you missing an assembly reference?) 
[D:\Repositories\corefx\src\shims\manual\System.Data.csproj]
```

To reproduce run:
```
build.cmd -os Unix
```
from Windows command line.

**Comment by davidsh** [comment]
@ViktorHofer

**Comment by danmosemsft** [comment]
What distro was this @ManickaP ?

**Comment by ManickaP** [comment]
No distro, ran on Windows for os `Unix`. 
I've learned later in #41029 that no such option exists but is one of the only two listed by builds help.

Apart from improving the help might also be nice to check the option value against supported list of OSes and report some understandable error.

This might be solved in #41029, closing.
